server/views/pages/immediateReleaseConditions.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{%  macro immediateReleaseConditions() %}
+    <p class="govuk-body">By default, the licence will automatically contain the following restrictions:</p>
+    <ul class="govuk-list govuk-list--bullet standard-conditions">
+        <li>standard conditions</li>
+        <li>the following additional condition:</li>
+        {{ govukInsetText({
+            text: "Not to approach or communicate with any victims of your offences without the prior approval of your supervising officer."
```

Comment:
Have we confirmed with UCD what the behaviour should be when a licence is PSS-only, or are we covering that in a separate ticket?

Reply:
Separate ticket

Reply:
It's CVSL-1691